project-metrics/metrics_service/env.py

```diff
@@ -0,0 +1,25 @@
+"""Provider of environment variables from env.yaml.
+
+Because app.yaml is only parsed when run on AppEngine, os.environ.get() will
+return None for key constants during local runs and testing. This module parses
+env.yaml directly and provides the environment variables, rather than using the
+env_variables part of app.yaml.
+"""
+
+import os
+from typing import Any, Mapping, Sequence, Text
+import yaml
+
+
+ENV_YAML_PATH = os.path.join(os.path.dirname(__file__), 'env.yaml')
+
+
+def _load_env_yaml_constants() -> Mapping[Text, Any]:
```

Comment:
This isn't memoize'd, so the file gets loaded and read every time you want to get an env variable...

Reply:
Good catch; added memoization.